HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -1,5 +1,54 @@
 import FWCore.ParameterSet.Config as cms
 
+# Updating the config (PR #8356)
+def customiseFor8356(process):
+    listL3trackingModule = ['hltL3MuonsOIState','hltL3MuonsOIHit','hltL3MuonsIOHit','hltL3NoFiltersNoVtxMuonsOIState','hltL3NoFiltersNoVtxMuonsOIHit','hltL3NoFiltersNoVtxMuonsIOHit']
+    for l3module in listL3trackingModule:
+        if hasattr(process, l3module):
+            if hasattr(getattr(process, l3module), 'GlbRefitterParameters'):
+                setattr(getattr(process, l3module).GlbRefitterParameters, 'RefitFlag', cms.bool(True))
+            if hasattr(getattr(process, l3module), 'L3TrajBuilderParameters'):
+                if hasattr(getattr(process, l3module).L3TrajBuilderParameters, 'MuonTrackingRegionBuilder'):
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'Pt_fixed', cms.bool(False))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'Z_fixed', cms.bool(True))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'Phi_fixed', cms.bool(False))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'Eta_fixed', cms.bool(False))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'Pt_min', cms.double(1.5))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'DeltaZ', cms.double(15.9))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'DeltaEta', cms.double(0.2))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'DeltaPhi', cms.double(0.2))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'maxRegions', cms.int32(1))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'precise', cms.bool(True))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'OnDemand', cms.int32(-1))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'input', cms.InputTag('hltL2Muons','UpdatedAtVtx'))
+                    setattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'MeasurementTrackerName', cms.InputTag("hltESPMeasurementTracker"))
+                    delattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'DeltaZ_Region')
+                    delattr(getattr(process, l3module).L3TrajBuilderParameters.MuonTrackingRegionBuilder, 'UseFixedRegion')
```

Comment:
Why are there no values assigned to 'DeltaZ_Region' and to 'UseFixedRegion' ?
Fix that here and below. Since it looks like the PSet MuonTrackingRegionBuilder 
is the same anyway, define that PSet as a python variable first, and then setattr
muonTrackingRegionBuilder to be that PSet for the various instances.